cms/static/js/views/video_thumbnail.js

```diff
@@ -64,13 +70,49 @@ define(
                         videoId: this.model.get('edx_video_id'),
                         actionInfo: this.actionsInfo[this.action],
                         thumbnailURL: this.model.get('course_video_image_url') || this.defaultVideoImageURL,
-                        duration: this.getDuration(this.model.get('duration'))
+                        duration: this.getDuration(this.model.get('duration')),
+                        videoImageSupportedFileFormats: this.getVideoImageSupportedFileFormats(),
+                        videoImageMaxSize: this.getVideoImageMaxSize(),
+                        videoImageResolution: this.getVideoImageResolution()
                     })
                 );
                 this.hideHoverState();
                 return this;
             },
 
+            getVideoImageSupportedFileFormats: function() {
+                var supportedFormats = _.reject(_.keys(this.videoImageSettings.supported_file_formats), function(item) {
+                    // Don't show redundant extensions to end user.
+                    return item === '.bmp2' || item === '.jpeg';
+                }).sort();
+                return {
+                    humanize: supportedFormats.slice(0, -1).join(', ') + ' or ' + supportedFormats.slice(-1),
+                    machine: _.values(this.videoImageSettings.supported_file_formats).sort()
+                };
```

Comment:
Even though we send sorted format from backend, `reject(_.keys(this.videoImageSettings.supported_file_formats)` changes the array and returns unsorted array. Therefore we need to do this here.